app/script/backup/Error.js

```diff
@@ -63,10 +63,18 @@ window.z.backup = z.backup || {};
     }
   }
 
+  class IncompatiblePlatformError extends ImportError {
+    constructor(message = 'Backup created by incompatible platform') {
```

Comment:
We should make these strings translatable.

Reply:
These strings are just used internally. We don't show them to the user, so no translation needed.

Reply:
they are not used either way. what is shown, are localized versions. this string is just for logging (if used).